ChangeLog

```diff
@@ -9,9 +19,39 @@
   - `taskd.server`
   - `taskd.trust`
 
-The Taskwarrior build no longer requires GnuTLS.
+  The Taskwarrior build no longer requires GnuTLS.
+
+  Deep thanks to the following for contributions to this work:
+
+  - Akash Shanmugaraj
```

Comment:
I am not sure from which perspective this is written, but I think @djmitche deserves mention.

Reply:
Fair point!